InsiteTeamTask/InsiteTeamTask/src/app/app.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../app/api.service';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent implements OnInit {
+  title = 'InsiteTeamTask';
+  attendance;
```

Comment:
For future reference, you should be creating interfaces for each of the types you expect to receive from the server, and typing these variables to add some type-safety (one of the main reasons for using TypeScript). For the Attendance object, it would look something like this:
```
// In a file named attendance.ts
export interface Attendance {
    barcode: string;
    memberId: number;
}

// In your api.service.ts
public getAttendanceByGameAndSeason(gameId: number, seasonId: number): Observable<Attendance[]> {
    return this.http.get<Attendance[]>(`https://localhost:44376/api/${gameId}/${seasonId}`);
}

// In this component class
attendance: Attendance[];
...
this.api.getAttendanceByGameAndSeason(gameId, seasonId).subscribe(attendance => {
  this.attendance = attendance;
});
```